tests/run.sh

```diff
@@ -9,5 +9,17 @@ set -e
 # - SPARK_VERSION
 # Default values can be looked up in tests/utils/common.go
 pushd $TEST_DIR
-go test -v -count=1 -timeout 30m .
+
+if [ -n "$TEAMCITY_VERSION" ]; then
+  # Running in Teamcity, therefore we need to convert `go test` output accordingly
+  if command -v go-test-teamcity; then
+    go test -v -count=1 -timeout 30m . | go-test-teamcity
+  else
+    go test -v -count=1 -timeout 30m . | docker run -i xjewer/go-test-teamcity
```

Comment:
it would be really great to avoid this dependency. as we're already installing the needed library in `docker-builder`, shall we use it instead?

Reply:
we should just aim to output results using junit xml format that the majority of CI systems already support it.
TeamCity has a build feature around XML processing which we can use of instead adding dependencies here
![image](https://user-images.githubusercontent.com/598287/67265613-753e1300-f4ae-11e9-98f4-2aa2cd8e5fea.png)

Reply:
I removed the usage of the 3rd party docker image. As for junit xml format, I don't think `go test` supports it out of the box, so it would be changing one dependency for another. 

Also I heard from Uladzimir, that there is a plan to update Teamcity soon, so this workaround could be dropped in future.